target/linux/ramips/mt7621/base-files/lib/upgrade/platform.sh

```diff
@@ -70,7 +70,8 @@ platform_do_upgrade() {
 	netgear,r6350|\
 	netgear,r6700-v2|\
 	netgear,r7200|\
-	netgear,r6800|\
+	netgear,r6900-v2|\
+        netgear,r6800|\
```

Comment:
Broken indent, wrong sorting (sorting seems undefined in several places).

Reply:
I didn't realize these were sorted. I've fixed this now.